rotkehlchen/db/cache.py

```diff
@@ -26,6 +26,7 @@ class DBCacheStatic(Enum):
     LAST_CREATE_REMINDER_CHECK_TS: Final = 'last_create_reminder_check_ts'
     LAST_GRAPH_DELEGATIONS_CHECK_TS: Final = 'last_graph_delegations_check_ts'
     LAST_GNOSISPAY_QUERY_TS: Final = 'last_gnosispay_query_ts'
+    LAST_HISTORICAL_BALANCES_RECREATION_TS: Final = 'last_historical_balances_recreation_ts'
```

Comment:
So for caching anything we should do it per asset. Not just on a total. Though not sure if the approach of having another table is requird.